### ISSUE TITLE
Clean up silx io using fabio 0.9

### ISSUE THREAD

**Issue by vallsv** [silx-kit/silx#2497]
Some code can be removed.

**Comment by vallsv** [comment]
Maybe check silx convert `template, begin, end` input, it could be merge into fabio.

**Comment by t20100** [comment]
Make sure to update fabio minimal required version here and there (setup.py, requirements.txt, doc?...)

**Comment by vallsv** [comment]
Oeuf corse

**Comment by t20100** [comment]
Moved to next milestone, not critical and it does not harm to keep compatibility with older versions a bit longer.

**Comment by t20100** [comment]
Removed from milestone, no specific need and requiring `fabio` >= 0.9 means dropping support of debian 9 with official backports (where only `fabio` 0.7 is available).

**Comment by vallsv** [comment]
There is dup code with fabio for EDF reading. But that's detail.

**Comment by t20100** [comment]
Yes it's definitely not closed, but it's already been postponed a few time so I just remove the milestone.
